### ISSUE TITLE
services stuck in "activating" state forever , containers are stuck in "networking" state.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#2005]
Server version - v0.37.0-rc1
Rancher server with 3 ubuntu hosts.

I was executing test cases relating to services which were all working fine and after a point now I see that  all services are stuck in "activating" state and the  containers are stuck in "networking" state.

Following exception seen constantly in logs:

```
2015-09-09 19:36:29,054 DEBUG [:] [] [] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [DONE] process [containerevent.create:25479] on resource [5553]
2015-09-09 19:36:29,056 DEBUG [:] [] [] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [containerevent.create:25480] on resource [5554]
2015-09-09 19:36:29,071 DEBUG [f5289154-c72a-4c9f-b75c-776b3db59a5e:25480] [containerEvent:5554] [containerevent.create->(ContainerEventPreCreate)] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Running pre listener [ContainerEventPreCreate]
2015-09-09 19:36:29,072 DEBUG [f5289154-c72a-4c9f-b75c-776b3db59a5e:25480] [containerEvent:5554] [containerevent.create->(ContainerEventPreCreate)] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Finished pre listener [ContainerEventPreCreate]
2015-09-09 19:36:29,078 DEBUG [f5289154-c72a-4c9f-b75c-776b3db59a5e:25480] [containerEvent:5554] [containerevent.create->(ContainerEventCreate)] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Running handler [ContainerEventCreate]
2015-09-09 19:36:29,079 DEBUG [f5289154-c72a-4c9f-b75c-776b3db59a5e:25480] [containerEvent:5554] [containerevent.create->(ContainerEventCreate)] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Finished handler [ContainerEventCreate]
2015-09-09 19:36:29,080 DEBUG [f5289154-c72a-4c9f-b75c-776b3db59a5e:25480] [containerEvent:5554] [containerevent.create->(ActivateByDefault)] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Running post listener [ActivateByDefault]
2015-09-09 19:36:29,080 DEBUG [f5289154-c72a-4c9f-b75c-776b3db59a5e:25480] [containerEvent:5554] [containerevent.create->(ActivateByDefault)] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Finished post listener [ActivateByDefault]
2015-09-09 19:36:29,086 INFO  [f5289154-c72a-4c9f-b75c-776b3db59a5e:25480] [containerEvent:5554] [containerevent.create] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Changing state [creating->created] on [containerEvent:5554]
2015-09-09 19:36:29,098 DEBUG [:] [] [] [] [cutorService-12] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [DONE] process [containerevent.create:25480] on resource [5554]
2015-09-09 19:36:29,592 INFO  [:] [] [] [] [p1843470554-504] [i.c.p.c.s.impl.ConfigItemServerImpl ] Setting item [host-routes] to version [1662-43fd6287135af077f5dffa7a2b5721a16cf12fe8b28a9e1bf522b35d19a80ea8] for [agent:3]
2015-09-09 19:36:29,610 INFO  [:] [] [] [] [ecutorService-3] [i.c.p.c.v.impl.ConfigUpdatePublisher]             Finished work item agent:3 [1412112813]
2015-09-09 19:36:29,610 INFO  [:] [] [] [] [ecutorService-3] [i.c.p.c.v.impl.ConfigUpdatePublisher]             Finished work item agent:3 [1538265094]
2015-09-09 19:36:29,644 INFO  [:] [] [] [] [p1843470554-505] [i.c.p.c.s.impl.ConfigItemServerImpl ] Processing [host-iptables] for client [agent:3]
2015-09-09 19:36:29,716 INFO  [:] [] [] [] [p1843470554-511] [i.c.p.c.s.impl.ConfigItemServerImpl ] Setting item [host-iptables] to version [3484-d049e47d422e7c2556577ff35624f118e182ec70793d23712724afa2f3165683] for [agent:3]
2015-09-09 19:36:29,733 INFO  [:] [] [] [] [ecutorService-3] [i.c.p.c.v.impl.ConfigUpdatePublisher]             Finished work item agent:3 [91482170]
2015-09-09 19:36:29,733 INFO  [:] [] [] [] [ecutorService-3] [i.c.p.c.v.impl.ConfigUpdatePublisher]             Finished work item agent:3 [2090023585]
2015-09-09 19:36:29,949 INFO  [:] [] [] [] [utorService-273] [i.c.p.e.a.MethodInvokingListener    ] Failed to acquire lock [SERVICE.414] for event [EventVO [id=93dc05ca-daf2-4b4d-9ecb-1c69e8b26a43, name=service.update, previousNames=null, replyTo=reply.5656
938958675457321, resourceId=414, resourceType=service, publisher=null, transitioning=null, transitioningMessage=null, transitioningInternalMessage=null, previousIds=null, data={configUrl=http://172.17.0.1:8080/v1, items=[{name=reconcile, requestedVersion=nu
ll, apply=false, increment=false, checkInSyncOnly=true}]}, time=Wed Sep 09 19:36:07 UTC 2015, listenerKey=null, transitioningProgress=null]]
2015-09-09 19:36:29,967 INFO  [:] [] [] [] [ecutorService-3] [i.c.p.c.v.impl.ConfigUpdatePublisher]             Finished work item service:265 [728304937]
2015-09-09 19:36:29,967 INFO  [:] [] [] [] [ecutorService-3] [i.c.p.c.v.impl.ConfigUpdatePublisher]             Finished work item service:265 [2111771906]
2015-09-09 19:36:30,957 ERROR [:] [] [] [] [utorService-140] [.p.c.v.i.ConfigItemStatusManagerImpl] Failed null, exit code [143] output [/var/lib/cattle/common/scripts.sh: line 62: 21813 Terminated              ${CATTLE_HOME}/config.sh "$@"
]
2015-09-09 19:36:30,964 ERROR [:] [] [] [] [utorService-281] [o.a.c.m.context.NoExceptionRunnable ] Uncaught exception java.lang.IllegalStateException: Failed to do service reconcile for service [269]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl$1.doWithLock(DeploymentManagerImpl.java:127) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl$1.doWithLock(DeploymentManagerImpl.java:100) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl$3.doWithLock(AbstractLockManagerImpl.java:40) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.LockManagerImpl.doLock(LockManagerImpl.java:33) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:13) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:37) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.activate(DeploymentManagerImpl.java:100) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.activate(DeploymentManagerImpl.java:83) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl$4.run(DeploymentManagerImpl.java:303) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.reconcileForClient(DeploymentManagerImpl.java:311) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.serviceUpdate(DeploymentManagerImpl.java:298) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at sun.reflect.GeneratedMethodAccessor1532.invoke(Unknown Source) ~[na:na]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_79]
        at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_79]
        at io.cattle.platform.eventing.annotation.MethodInvokingListener$1.doWithLockNoResult(MethodInvokingListener.java:76) ~[cattle-framework-eventing-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.LockCallbackNoReturn.doWithLock(LockCallbackNoReturn.java:7) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.LockCallbackNoReturn.doWithLock(LockCallbackNoReturn.java:3) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl$3.doWithLock(AbstractLockManagerImpl.java:40) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.LockManagerImpl.doLock(LockManagerImpl.java:33) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:13) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:37) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.eventing.annotation.MethodInvokingListener.onEvent(MethodInvokingListener.java:72) ~[cattle-framework-eventing-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.eventing.impl.AbstractThreadPoolingEventService$2.doRun(AbstractThreadPoolingEventService.java:135) ~[cattle-framework-eventing-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.NoExceptionRunnable.runInContext(NoExceptionRunnable.java:13) ~[cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.ManagedContextRunnable$1.run(ManagedContextRunnable.java:49) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.impl.DefaultManagedContext$1.call(DefaultManagedContext.java:55) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.impl.DefaultManagedContext.callWithContext(DefaultManagedContext.java:108) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.impl.DefaultManagedContext.runWithContext(DefaultManagedContext.java:52) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.ManagedContextRunnable.run(ManagedContextRunnable.java:46) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_79]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_79]
        at java.lang.Thread.run(Thread.java:745) [na:1.7.0_79]

2015-09-09 19:36:31,123 ERROR [:] [] [] [] [utorService-150] [.p.c.v.i.ConfigItemStatusManagerImpl] Failed null, exit code [143] output [/var/lib/cattle/common/scripts.sh: line 62: 19707 Terminated              ${CATTLE_HOME}/config.sh "$@"
]
2015-09-09 19:36:31,134 ERROR [:] [] [] [] [utorService-140] [.p.c.v.i.ConfigItemStatusManagerImpl] Failed null, exit code [143] output [/var/lib/cattle/common/scripts.sh: line 62: 20070 Terminated              ${CATTLE_HOME}/config.sh "$@"
]
2015-09-09 19:36:31,192 ERROR [:] [] [] [] [utorService-275] [.p.c.v.i.ConfigItemStatusManagerImpl] Failed null, exit code [143] output [/var/lib/cattle/common/scripts.sh: line 62: 20074 Terminated              ${CATTLE_HOME}/config.sh "$@"
]
2015-09-09 19:36:31,199 ERROR [:] [] [] [] [utorService-275] [.p.c.v.i.ConfigItemStatusManagerImpl] Failed null, exit code [143] output [/var/lib/cattle/common/scripts.sh: line 62: 19695 Terminated              ${CATTLE_HOME}/config.sh "$@"
]
2015-09-09 19:36:31,950 INFO  [:] [] [] [] [utorService-140] [i.c.p.e.a.MethodInvokingListener    ] Failed to acquire lock [SERVICE.414] for event [EventVO [id=93dc05ca-daf2-4b4d-9ecb-1c69e8b26a43, name=service.update, previousNames=null, replyTo=reply.5656
938958675457321, resourceId=414, resourceType=service, publisher=null, transitioning=null, transitioningMessage=null, transitioningInternalMessage=null, previousIds=null, data={configUrl=http://172.17.0.1:8080/v1, items=[{name=reconcile, requestedVersion=nu
ll, apply=false, increment=false, checkInSyncOnly=true}]}, time=Wed Sep 09 19:36:07 UTC 2015, listenerKey=null, transitioningProgress=null]]
2015-09-09 19:36:32,022 ERROR [:] [] [] [] [utorService-281] [.p.c.v.i.ConfigItemStatusManagerImpl] Failed null, exit code [137] output [/var/lib/cattle/common/scripts.sh: line 62: 21854 Killed                  ${CATTLE_HOME}/config.sh "$@"
]
2015-09-09 19:36:32,956 ERROR [:] [] [] [] [utorService-140] [o.a.c.m.context.NoExceptionRunnable ] Uncaught exception java.lang.IllegalStateException: Failed to do service reconcile for service [269]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl$1.doWithLock(DeploymentManagerImpl.java:127) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl$1.doWithLock(DeploymentManagerImpl.java:100) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl$3.doWithLock(AbstractLockManagerImpl.java:40) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.LockManagerImpl.doLock(LockManagerImpl.java:33) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:13) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:37) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.activate(DeploymentManagerImpl.java:100) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.activate(DeploymentManagerImpl.java:83) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl$4.run(DeploymentManagerImpl.java:303) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.reconcileForClient(DeploymentManagerImpl.java:311) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.servicediscovery.deployment.impl.DeploymentManagerImpl.serviceUpdate(DeploymentManagerImpl.java:298) ~[cattle-iaas-service-discovery-server-0.5.0-SNAPSHOT.jar:na]
        at sun.reflect.GeneratedMethodAccessor1532.invoke(Unknown Source) ~[na:na]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_79]
        at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_79]
        at io.cattle.platform.eventing.annotation.MethodInvokingListener$1.doWithLockNoResult(MethodInvokingListener.java:76) ~[cattle-framework-eventing-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.LockCallbackNoReturn.doWithLock(LockCallbackNoReturn.java:7) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.LockCallbackNoReturn.doWithLock(LockCallbackNoReturn.java:3) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl$3.doWithLock(AbstractLockManagerImpl.java:40) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.LockManagerImpl.doLock(LockManagerImpl.java:33) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:13) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl.lock(AbstractLockManagerImpl.java:37) ~[cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.eventing.annotation.MethodInvokingListener.onEvent(MethodInvokingListener.java:72) ~[cattle-framework-eventing-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.eventing.impl.AbstractThreadPoolingEventService$2.doRun(AbstractThreadPoolingEventService.java:135) ~[cattle-framework-eventing-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.NoExceptionRunnable.runInContext(NoExceptionRunnable.java:13) ~[cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.ManagedContextRunnable$1.run(ManagedContextRunnable.java:49) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.impl.DefaultManagedContext$1.call(DefaultManagedContext.java:55) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.impl.DefaultManagedContext.callWithContext(DefaultManagedContext.java:108) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.impl.DefaultManagedContext.runWithContext(DefaultManagedContext.java:52) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at org.apache.cloudstack.managed.context.ManagedContextRunnable.run(ManagedContextRunnable.java:46) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_79]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_79]
        at java.lang.Thread.run(Thread.java:745) [na:1.7.0_79]
```

```
mysql> select id,name,state from instance where state="starting";   
+-----+-------------------------+----------+
| id  | name                    | state    |
+-----+-------------------------+----------+
| 518 | Lb Agent                | starting |
| 522 | test310103_test732207_1 | starting |
| 523 | test310103_test732207_2 | starting |
| 524 | Lb Agent                | starting |
| 551 | Lb Agent                | starting |
| 760 | test734223_test145809_2 | starting |
| 762 | test734223_test618103_3 | starting |
| 764 | test566521_test20048_1  | starting |
| 770 | test655901_test111160_3 | starting |
| 772 | test987325_test377598_1 | starting |
| 774 | test987325_test295932_2 | starting |
| 778 | test330938_test686542_2 | starting |
| 782 | test47821_test604663_2  | starting |
| 785 | test374329_test315109_1 | starting |
| 788 | Default_testclient_1    | starting |
| 792 | test588417_test692675_2 | starting |
| 794 | test756056_test812139_2 | starting |
| 797 | test187250_test737952_1 | starting |
| 798 | test187250_test353789_1 | starting |
| 804 | test850747_test125084_2 | starting |
| 807 | test136138_test686475_2 | starting |
| 811 | test480365_test752691_2 | starting |
| 813 | test55718_test378758_1  | starting |
| 816 | test21141_test763110_1  | starting |
+-----+-------------------------+----------+


mysql> select id,name,state from service where state in ("activating","updating-active");
+-----+---------------+-----------------+
| id  | name          | state           |
+-----+---------------+-----------------+
| 268 | test732207    | activating      |
| 269 | LB-test183082 | activating      |
| 394 | test145809    | activating      |
| 395 | test618103    | activating      |
| 397 | test20048     | activating      |
| 399 | test111160    | activating      |
| 400 | test377598    | activating      |
| 401 | test295932    | activating      |
| 403 | test686542    | activating      |
| 405 | test604663    | activating      |
| 407 | test315109    | activating      |
| 409 | testclient    | activating      |
| 411 | test692675    | activating      |
| 412 | test812139    | activating      |
| 414 | test737952    | activating      |
| 415 | test353789    | activating      |
| 416 | test125084    | activating      |
| 418 | test686475    | activating      |
| 420 | test752691    | activating      |
| 422 | test378758    | activating      |
| 424 | test763110    | activating      |
| 426 | test318323    | activating      |
| 427 | test727171    | activating      |
| 265 | LB-test610367 | updating-active |
| 284 | LB-test4186   | updating-active |
| 286 | LB-test62846  | updating-active |
| 425 | test604169    | updating-active |
+-----+---------------+-----------------+
27 rows in set (0.00 sec)
```

**Comment by pwFoo** [comment]
Tested with Rancher v0.35.0 release and v0.37.0-rc2 on RancherOS. Service is created by rancher-compose and stucks in networking.

mysql + mysql-storage containers working fine, but apache + apache-storage stucks in networking state.

rancher-compose.yml
Apache:
  scale: 1
Apache-Storage:
  scale: 1
MySQL:
  scale: 1
MySQL-Storage:
  scale: 1

docker-compose.yml
MySQL:
  name: MySQL
  ports:
  - 3306:3306/tcp
  restart: always
  tty: true
  image: centos/mariadb
  stdin_open: true
  labels:
    io.rancher.sidekicks: MySQL-Storage
  volumes-from:
  - MySQL-Storage
  environment:
  -  MYSQL_ROOT_PASSWORD="nwenwe"
MySQL-Storage:
  name: MySQL-Storage
  tty: true
  command:
  - cat
  image: centos/mariadb
  volumes:
  - /var/lib/mysql
  stdin_open: true
  environment:
  - MYSQL_ROOT_PASSWORD="nwenwe"
Apache:
  name: Apache
  ports:
  - 80:80/tcp
  restart: always
  tty: true
  image: centos/httpd
  stdin_open: true
  links:
  - MySQL:db01
  labels:
    io.rancher.sidekicks: Apache-Storage
  volumes-from:
  - Apache-Storage
Apache-Storage:
  name: Apache-Storage
  tty: true
  command:
  - cat
  image: centos/httpd
  volumes:
  - /var/www/httpd
  stdin_open: true

**Comment by sangeethah** [comment]
@pwFoo , The issue you are seeing seems to be different from the one i was  seeing. In my case , after I see the issue where LB agents get stuck in "networking" state , all services ended up being stuck in "activating" state and  containers are stuck in "networking" state .

This issue is fixed in  v0.37.0-rc2 and I was able to verify that this issue does not happen anymore in  v0.37.0-rc2 setup.

could you please log a different bug to track your issue ?
